### DISCUSSION TITLE
Disable zoom and enable only pan

### DISCUSSION THREAD

**Discussion by Kromzem** [Live-Charts/LiveCharts2#1121] [Q&A] [answered]
Hey :)

Is it possible to disable zooming completely and only allow panning? 
I've not been able to find how to do this 😄

**Comment by beto-rodriguez** [comment, marked as answer]
This is an interesting feature that is missing in the library, will move this to an issue to track the implementation of this feature.

**Reply by Kromzem** [reply]
Thank you so much for your fast reply :) Will looking forward to this feature ;)
